client/components/jetpack/intro-pricing-banner/index.tsx

```diff
@@ -91,13 +89,7 @@ const IntroPricingBanner: FunctionComponent< Props > = ( { productSlugs, siteId
 				) }
 				{ ( isNotNarrow || ( discountPercentage <= 0 && ! isLoading ) ) && (
 					<div className="intro-pricing-banner__guarantee">
-						<img
-							src={ guaranteeBadge }
-							alt={ translate( 'Money Back %(days)d-Day Guarantee Badge', {
-								args: { days: GUARANTEE_DAYS },
-								textOnly: true,
-							} ) }
-						/>
+						<img src={ guaranteeBadge } alt="" />
```

Comment:
> Shouldn't we leave the alt as it was for this image?

The text just after has the same description, so it would be read twice by a screen reader.

> If not, then probably we do not need to import GUARANTEE_DAYS

We do use it in the text right after.